src/GAP.jl

```diff
@@ -318,6 +320,25 @@ function randseed!(seed::Union{Integer,Nothing}=nothing)
     nothing
 end
 
+import Pkg
+
+"""
+    versioninfo(io::IO = stdout; jll::Bool = false)
+
+Print the version numbers of GAP.jl and GAP,
+and version numbers and installation paths of all currently loaded GAP packages.
+Note that these paths can be nonstandard because Julia's package manager
+does not control which available version of a GAP package gets loaded.
+
+If `jll` is `true` then also the underlying binary packages (jll),
+if available, are included in the output.
+"""
+function versioninfo(io::IO = stdout; jll::Bool = false, padding::String = "")
```

Comment:
Could you additionally add a kwarg `full::Bool = false` with the semantics as in `Oscar.versioninfo()`, i.e. `full=false` does nothing, but `full=true` sets all other kwargs (aka here just `jll`) to `true`?
This would then make it possible to ask users just for `GAP.versioninfo(full=true)` and this includes everything that is available (even if we only add it at some point in the future)